src/widget.rs

```diff
@@ -274,7 +286,7 @@ impl Widget for PseudoTerminal<'_> {
     }
 }
 
-#[cfg(test)]
+#[cfg(all(test, feature = "vt100"))]
```

Comment:
These tests could probably be moved to the `vt100_imp` module, but felt like some unnecessary movement for this PR.

Reply:
I agree.